lib/Mojo/Collection.pm

```diff
@@ -322,6 +324,16 @@ Alias for L<Mojo::Base/"tap">.
 
 Turn collection into array reference.
 
+=head2 TO_JSON
+
+  my $array = $collection->TO_JSON;
+
+Turn collection into array reference allowing conversion to JSON by
+L<Mojo::JSON>.
```

Comment:
It is merely an alias for `to_array`, that's what the description should say (and link to).